src/msw/menu.cpp

```diff
@@ -916,30 +922,47 @@ void wxMenuBar::Refresh()
 
 WXHMENU wxMenuBar::Create()
 {
-    if ( m_hMenu != 0 )
+    if (m_hMenu != 0)
         return m_hMenu;
 
     m_hMenu = (WXHMENU)::CreateMenu();
 
-    if ( !m_hMenu )
+    if (!m_hMenu)
     {
         wxLogLastError(wxT("CreateMenu"));
     }
     else
     {
-        for ( wxMenuList::iterator it = m_menus.begin();
-              it != m_menus.end();
-              ++it )
+        for (wxMenuList::iterator it = m_menus.begin();
+            it != m_menus.end();
+            ++it)
         {
-            if ( !::AppendMenu((HMENU)m_hMenu, MF_POPUP | MF_STRING,
-                               (UINT_PTR)(*it)->GetHMenu(),
-                               (*it)->GetTitle().t_str()) )
+            if (!::AppendMenu((HMENU)m_hMenu, MF_POPUP | MF_STRING,
+                (UINT_PTR)(*it)->GetHMenu(),
+                (*it)->GetTitle().t_str()))
             {
                 wxLogLastError(wxT("AppendMenu"));
             }
         }
-    }
 
+#if wxUSE_OWNER_DRAWN
+        if (wxMSWDarkMode::IsActive())
+        {
+            // Set dark mode menu background color
+            // May be this should be done in wxMenu::Init() instead?
+            wxUxThemeHandle hTheme(GetFrame()->AsWindow(), L"MENU", L"DarkMode_ImmersiveStart::Menu");
+            COLORREF crMenu = wxColourToRGB(hTheme.GetColour(MENU_POPUPBACKGROUND, TMT_FILLCOLOR));
```

Comment:
I am not sure about getting the color from the theme (here and elsewhere): How does `wxDarkModeSettings::GetColour()` is expected to be used?